src/stores/media/index.ts

```diff
@@ -144,7 +144,15 @@ export const useMediaStore = defineStore('media', {
       }
     },
 
-    allMedia(): Media[] {
+    /**
+     * Returns a mixed bag of search results across media types.
+     *
+     * This does not contain all hits across all media types! It contains all
+     * image results and as many audio results as can be sparsely spliced into
+     * the image results. The leftover audio results will appear when more
+     * results are requested using the 'Load more' button.
+     */
+    allMedia(state): Media[] {
```

Comment:
Few small things:

- 'hits' might be clearer than 'results' language-wise
- Is this inaccurate, should it be 'media with the most results' vs. and as many 'other' results, given we'll have other media types soon?

Reply:
Updated documentation in 407d9b4.